README.md

```diff
@@ -1,54 +1,91 @@
 
+* **Static route** - A static route will render a view that does not change, you typically will not send parameters to it, examples would be: a site's about or contact pages.
+* **Dynamic route** - Dynamic routes are pages that accept parameters and render different content based on those parameters, an example would be: a blog's post page that contains a specific article.
 
-## Notes
+In this lesson we are going to specifically cover static pages to ensure that you can get a firm understanding of how routing works in a Rails application.
 
-Should explain that every URL that we want our application to handle must map to a specific controller and action.
+Before we dive into the code and routing configurations, it helps to know how HTTP works at a high level. Below is the flow that takes place when a user attempts to go to a page on a Rails application:
 
-When a request comes into our application, our router must pick a controller and a method in that controller that is responsible for generating a response. We've taught a lot of this in Sinatra so it can be a bit light here.
+1. A URL is entered into the browser, this is the HTTP request
+2. That request is sent to the server where the application's router interprets the request and sends a message to the controller mapped to that route
+3. The controller communicates with the view file mapped to the controller method
+4. The server returns that HTTP response, which contains the view and page that can be viewed in the browser
 
-Use the example of how we want to have an 'about' page on our site. So let's try to go there, let's just type localhost:3000/about.
+## Implementing a Static Route
 
-We get a routing error. Let's draw the route.
+Let's try this out in our application, I'm going to continue using the ```blog-flash``` application that I walked through in the Rails Basic lesson.
 
-Let's draw a simple route, like /about. Let's have that map to a StaticController and an about action (though don't create that controller). Reload the page. No more routing error, now it can't find the controller.
+To begin, startup the rails server and go to ```localhost:3000/about``` as you will see this throws a routing error: ```No route matches [GET] "/about"```, to fix this, stop the rails server by pressing ```control + c``` (anytime you make a routing change you need to restart the rails server)
 
-Let's make a new controller, no generators, with touch app/controllers/static_controller.rb.
+Now draw the route by opening the ```config/routes.rb``` file and add the following route inside of the ```draw``` block:
 
-Controllers need to inherit from ApplicationController and explain briefly the controller naming convention.
+```ruby
+get 'about', to: 'static#about'
```

Comment:
Can you break down the three parts of this? The verb (they know HTTP verbs) the path (they know paths) and then the `static#about` They know MVC.